source/include/mqtt_agent_command_functions.h

```diff
@@ -195,7 +195,7 @@ MQTTStatus_t MQTTAgentCommand_Ping( MQTTAgentContext_t * pMqttAgentContext,
  * @param[in] pUnusedArg Unused NULL argument.
  * @param[out] pReturnFlags Flags set to indicate actions the MQTT agent should take.
  *
- * @return `MQTTSuccess`.
+ * @return #MQTTSuccess.
```

Comment:
How about references to the coreMQTT API functions (like `MQTT_Ping` in `MQTTAgentCommand_Ping` function documentation)?

Reply:
Doxygen's smart enough to automatically link functions and data types without needing the `#`. It's just for specific enum values like this we need it